openlibrary/templates/site/body.html

```diff
@@ -31,10 +31,36 @@
       </div>
       $# Paste next line into announcement variable to show the blue banner
       $# <strong>Title:</strong> Description &nbsp; <a href="#" data-ol-link-track="AnnouncementsBar|BlogWidget" class="btn primary">Learn More</a>
-      $ announcement = None
-      $if announcement:
-        <div class="page-banner page-banner-body">
-         $:announcement
+      $# Please keep the announcement a dict object with 'title' and 'message' keys
+      $ announcement = {'title': None, 'message': None}
+      $if announcement and announcement['message'] and announcement['title']:
```

Comment:
```suggestion
      $if announcement['message'] and announcement['title']:
```

Reply:
hmm I added it  just in case the `announcement` variable is accidentally set to `None`. It's an unlikely case that such error will make it to production but i thought it would be a good precaution.